SchedulerJobs/SchedulerJobs.Services/VideoApiService.cs

```diff
@@ -11,11 +11,9 @@ namespace SchedulerJobs.Services
 {
     public interface IVideoApiService
     {
-        Task<List<ConferenceSummaryResponse>> GetOpenConferencesByScheduledDate(DateTime scheduledDate);
-        
-        Task CloseConference(Guid conferenceId);
+        Task<List<ConferenceSummaryResponse>> GetExpiredOpenConferences();
```

Comment:
The Contract models in 'SchedulerJobs.Services' are incorrect. You simply need [ExpiredConferencesResponse.cs](https://github.com/hmcts/vh-video-api/blob/master/VideoAPI/VideoApi.Contract/Responses/ExpiredConferencesResponse.cs) from VideoApi

R

Reply:
Update model to ExpiredConferencesResponse.cs

Reply:
There is no need for participant summary response. Could you please check there are no additional files that are not needed?